plugins/source/aws/docs/tables/aws_directconnect_locations.md

```diff
@@ -0,0 +1,21 @@
+# Table: aws_directconnect_locations
+
+https://docs.aws.amazon.com/directconnect/latest/APIReference/API_Connection.html
+
+The primary key for this table is **_cq_id**.
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|region|String|
+|available_mac_sec_port_speeds|StringArray|
+|available_port_speeds|StringArray|
+|available_providers|StringArray|
+|location_code|String|
```

Comment:
Do we know if the `account_id,region,location_code,location_name` combination is unique and can be used as a PK?

Reply:
All of the other fields are attributes of a location... I would also be alright just hashing the entire object because it is 100% AWS defined without any user input

Reply:
I think we can keep this for now and change based on users feedback

Reply:
Seems like it's worth using `account_id,region,location_code`, if all the other fields are attributes of the location. Then this table will also work fine with `overwrite` mode